findDuplicates.py

```diff
@@ -38,30 +41,36 @@ def output(text=None, level=0, end="\n", flush=False):
         print(text, end=end, flush=flush)
 
 
-def search_pattern(file_type):
-    return "*." + file_type
-
+def search_pattern(file_types):
+    pattern = '([^.].*)[.](' + file_types[0]
```

Comment:
I've been using Black to format. Can you please use double quotes for consistency?

```suggestion
    pattern = "([^.].*)[.](" + file_types[0]
```